### PR TITLE
Revolver: Minimal integration example

### PR DESCRIPTION
**Pull request by alsedi**
This is the beginning of the examples section. I'm planning to add more examples one by one, covering iOS, Android and KMP integrations. However, it takes a lot of time on commenting the code and reveal features in proper order.

### CODE REVIEW

File: examples/README.md
Comment (TheRogue76) [inline examples/README.md:1]: Maybe add something or delete this file? Honestly not sure. Could be a link to the other readmes and act as a directory

Comment (alsedi) [reply examples/README.md:1]: yes, this will be a home for all examples. I forgot to add the content

**Review by jacksonmafra-umain — APPROVED** [review]

**Review by casvanluijtelaar — COMMENTED** [review]
This is good, but ideally it should be a small KMP project, not just an android app. like one of those where ios, android and kmp exist in. the same folder. then we can create an example project with one vm for both platforms

**Comment by alsedi** [comment]
yes, I will add it also. SO it will be Android + iOS + KMM example

**Comment by alsedi** [comment]
Idea is to show that Revolver is easy to use and integrate in any case and if developers do not want to run KMP project they may skip it and use Revolver directly on the platform.

**Pull request merged**